lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -1337,8 +1356,8 @@ public function getGoogleKey()
     protected function readPrintCapabilities(QgisProject $qgsLoad)
     {
         $printTemplates = array();
-        $options = $this->getOptions();
-        if ($options && property_exists($options, 'print') && $options->print == 'True') {
+        $print = $this->cfg->getOption('print');
+        if ($print == 'True') {
```

Comment:
```suggestion
        if ($this->optionToBoolean($print)) {
```
A suggestion, not a recommandation